cfme/services/catalogs/catalog_item.py

```diff
@@ -42,23 +44,77 @@ class RequestInfoTab(Tab, BasicProvisionFormView):
     TAB_NAME = 'Request Info'
 
 
+class IconItem(ParametrizedView, ClickableMixin):
+    PARAMETERS = ('icon_title',)
+    ROOT = ParametrizedLocator('//div[@class = "tab-content"]//span/i[contains('
+                               '@title, {icon_title|quote})]')
+
+
+class ButtonImageSelectView(View):
```

Comment:
**QUESTION**
Does that widget https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L3603 do the same?

Reply:
Yes it is, @quarckster thanks a lot. I deleted unnecessary code